_staff_members/subham-rai.md

```diff
@@ -2,6 +2,6 @@
 name: Subham Rai
 position: Contributor
 image_path: /images/photos/subham-rai.jpg
-# twitter: 
+twitter: https://github.com/subhamkrai
```

Comment:
@gunjanmimo, I might be missing it, but I still see "twitter: " as the label used for what is a GitHub.com reference to @subhamkrai's profile. This needs changed to "github: ".